src/collective/instancebehavior/tests/test_instancebehavior.py

```diff
@@ -1,4 +1,5 @@
-import unittest2 as unittest
+# -*- coding: utf-8 -*-
```

Comment:
@gforcada if you add non ASCII characters to source without setting this header you'll get a syntax error. Anyway, nevermind ;)